### PR TITLE
Text improvement suggestions

### PR DESCRIPTION
**Pull request by ItachiSan**
Some small suggestions for the text.

As they are "suggestions", these can be completely accepted, discussed or dropped.
I thought of those while translating the strings to Italian.

All is good, however I would like some chat on dropped strings :)

### CODE REVIEW

File: en-GB/Settings.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ export default {
       'Description': 'Stop & Start the bot'
     },
     'TwitchBotCommands': {
-      'Title': 'Customise the Bot Command names',
+      'Title': 'Customise Bot Command',
```
Comment (Tumbledore) [inline en-GB/Settings.js]: shouldn't this be like `Commands` because these are plural? because it was `names` before that

File: en-GB/Setup.js [hunk 2]
```diff
@@ -55,20 +55,20 @@ export default {
...
     'WaitingForGameData': 'Waiting for Game data...',
     'ConnectingToEmulator': 'Attempting to automatically connect to {emulator}',
     'LoadLuaScriptIntoEmulator': 'Load the Lua Script into your emulator',
     'Instructions': {
-      'VBA-RR': 'This can be done by going to <em>Tools</em> » <em>Lua Script Window</em> » <em>New Lua Script Window</em> inside VBA-ReRecording <em>(Be sure not to close the script window, you can minimize it though)</em',
-      'DeSmuME': 'This can be done by going to <em>Tools</em> » <em>Lua Scripting</em> » <em>New Lua Script Window</em> inside of <em>DeSmuME</strong> <em>(Be sure not to close the script window, you can minimize it though)</em>'
+      'VBA-RR': 'This can be done by going to <em>Tools</em> » <em>Lua Script Window</em> » <em>New Lua Script Window</em> inside <em>VBA-ReRecording</em> <em>(Be sure not to close the script window, you can minimize it though)</em',
+      'DeSmuME': 'This can be done by going to <em>Tools</em> » <em>Lua Scripting</em> » <em>New Lua Script Window</em> inside of <em>DeSmuME</em> <em>(Be sure not to close the script window, you can minimize it though)</em>'
```
Comment (Tumbledore) [inline en-GB/Setup.js:71]: i don't know how these `<em>`-Tags are used later on, but wouldn't one opening and closing Tag be enough there? like `<em>DeSmuME (Be sure not to close the script window, you can minimize it though)</em>`

Comment (Tumbledore) [reply en-GB/Setup.js:71]: same for `VBA-ReRecording` as well

File: en-GB/Settings.js (hunk 1 above)
Comment (ItachiSan) [reply en-GB/Settings.js]: Agreed on that! 😄

File: en-GB/Setup.js (hunk 2 above)
Comment (ItachiSan) [reply en-GB/Setup.js:71]: They are for styling (they make bold text in HTML), my guess is that those get rendered in the app.

About incorporating the last part in a single tag, I understand but I am not super sure.
It does the same, however it logically becomes a full block.

Except this single point, I agree and have no issue changing that. 🙂

Comment (NoelDavies) [reply en-GB/Setup.js:71]: Having tags in language strings isn't perfect, I know. They are going to be removed at a later point.

File: en-GB/Settings.js (hunk 1 above)
Comment (NoelDavies) [reply en-GB/Settings.js]: Oh God, how did I miss this?!

**Pull request merged**